### PR TITLE
Ep 3174 collection actions

### PR DESCRIPTION
**Pull request by ancashoria**

### CODE REVIEW

File: packages/earth-map/src/components/collection/collection-delete/CollectionDelete.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      </div>
+      {saveError && <p className="ng-form-error-block ng-margin-bottom">{saveError}</p>}
+    </Modal>
+  );
+
+  async function handleDelete() {
+    try {
+      await deleteCollection(id, { group: organization });
+
+      dispatch({
```
Comment (corinacioloca) [inline packages/earth-map/src/components/collection/collection-delete/CollectionDelete.tsx:67]: why aren't you using the actions from redux?

Comment (corinacioloca) [reply packages/earth-map/src/components/collection/collection-delete/CollectionDelete.tsx:67]: and you can group them in index when you connect the component

File: packages/earth-map/src/components/collection/collection-details/CollectionDetails.tsx [hunk 2]
```diff
@@ -110,92 +126,24 @@ const CollectionDetails = (props: IProps) => {
...
+        <CollectionEditPlaces
+          collection={data}
+          placesFromGroups={placesFromGroups}
+          setCollectionData={setCollectionData}
+          setMapBounds={setMapBounds}
+          toggleEditPlaces={toggleEditPlaces}
+        />
       )}
+
+      <CollectionDelete collection={data} isDeleting={isDeleting} setIsDeleting={setIsDeleting} />
```
Comment (corinacioloca) [inline packages/earth-map/src/components/collection/collection-details/CollectionDetails.tsx]: mount this when it goes into delete state

File: packages/earth-map/src/components/collection/collection-delete/CollectionDelete.tsx (hunk 1 above)
Comment (ancashoria) [reply packages/earth-map/src/components/collection/collection-delete/CollectionDelete.tsx:67]: That was my goal too, but I ran into some issues with actions not being fired. We can take a look

**Pull request merged**